### DISCUSSION TITLE
🗜 效率工具使用

### DISCUSSION THREAD

**Discussion by giscus** [oldwinter/knowledge-garden#10] [备发布站的评论系统]
# 🗜 效率工具使用

up:: [[☘️ 花园导览 🍀]]x:: [[+ 平台+插件]]

https://notes.oldwinter.top/%E6%95%88%E7%8E%87%E5%B7%A5%E5%85%B7%E4%BD%BF%E7%94%A8

**Comment by oldwinter** [comment]
测试一下

**Comment by neverup7** [comment]
补充一个浏览器插件推荐
详情:https://github.com/neverp/chajian
![](https://files.catbox.moe/00s617.png)
